### PR TITLE
Ssl certificate issues

### PR DESCRIPTION
**Pull request by ylow**
- Some code cleanup (isolates curl option setting code into a single file)
 - added option documentation to get_runtime_config()
 - Added an option to disable all certificate checks. (This is the ultimate fallback option. It also works for accessing files from https)

### CODE REVIEW

File: oss_src/fileio/fileio_constants.cpp [hunk 1]
```diff
@@ -162,6 +164,10 @@ const std::string& get_alternative_ssl_cert_file() {
   return FILEIO_ALTERNATIVE_SSL_CERT_FILE;
 }
 
+const bool disable_ssl_cert_checks() {
+  return FILEIO_DISABLE_SSL_CERTIFICATE_CHECKS != 0;
+}
```
Comment (esamanas) [inline oss_src/fileio/fileio_constants.cpp]: eh...does this tell you if ssl cert checks are disabled? Weird name for it, if so.

File: oss_src/fileio/fileio_constants.cpp [hunk 2]
```diff
@@ -151,8 +151,10 @@ EXPORT std::string FILEIO_ALTERNATIVE_SSL_CERT_FILE = "/etc/pki/tls/certs/ca-bun
 EXPORT std::string FILEIO_ALTERNATIVE_SSL_CERT_DIR = "";
 EXPORT std::string FILEIO_ALTERNATIVE_SSL_CERT_FILE = "";
 #endif
+EXPORT int64_t FILEIO_DISABLE_SSL_CERTIFICATE_CHECKS = 0;
 REGISTER_GLOBAL(std::string, FILEIO_ALTERNATIVE_SSL_CERT_FILE, true);
 REGISTER_GLOBAL(std::string, FILEIO_ALTERNATIVE_SSL_CERT_DIR, true);
+REGISTER_GLOBAL(int64_t, FILEIO_DISABLE_SSL_CERTIFICATE_CHECKS, true);
```
Comment (esamanas) [inline oss_src/fileio/fileio_constants.cpp]: so...are they disabled by default or not?

File: oss_src/unity/python/sframe/util/__init__.py [hunk 3]
```diff
@@ -380,14 +380,11 @@ def get_runtime_config():
 
 def set_runtime_config(name, value):
     """
-    Sets a runtime configuration value. These configuration values are also
+    Configures sytem behavior at runtime. These configuration values are also
```
Comment (esamanas) [inline oss_src/unity/python/sframe/util/__init__.py]: Typo

File: oss_src/fileio/fileio_constants.cpp (hunk 1 above)
Comment (ylow) [inline oss_src/fileio/fileio_constants.cpp]: yeah. True if disabled... slightly funky yes. Maybe I will change disable to INSECURE

**Pull request merged**